lib/td/command/export.rb

```diff
@@ -75,6 +79,7 @@ def table_export(op)
     s3_opts['access_key_id'] = aws_access_key_id
     s3_opts['secret_access_key'] = aws_secret_access_key
     s3_opts['pool_name'] = pool_name if pool_name
+    s3_opts['encryption'] = encryption if encryption
```

Comment:
we can validate the value of encryption on client-side before sending the job task to td-api.